.github/workflows/tests.yaml

```diff
@@ -22,10 +22,11 @@ jobs:
             matrix:
                 php: ['8.1']
                 path:
-                    - tests
-                    - rules-tests
-                    - packages-tests
-                    - utils-tests
+                    - "tests/"
+                    - "--testsuite p-rules"
```

Comment:
Could you update name some longer more descriptive one like "php-upgrade-rules"? The "p" could be anything here

Reply:
p means start from "P", if it goes to "php-upgrade-rules", the pattern need to be updated in phpunix.xml to be:

```php
    <testsuite name="php-upgrade-rules">
        <directory>rules-tests/Php*</directory>
    </testsuite>
    <testsuite name="other-rules">
        <directory>rules-tests/</directory>
        <exclude>rules-tests/Php*</exclude>
    </testsuite>
```

Reply:
@samsonasik your suggestions is what I had before. I reduced it to "P" only, since the group without "P" was a bit too big. using only "P" the balance of the 2 groups was better

Reply:
While that makes performance better, it's unclear what P means without more context. The DX is also improtant for further maintenance.

Could you split it into another groups that have descriptive name?

Reply:
I couldn't come up with better named group

Reply:
I'd go with these for now: https://github.com/rectorphp/rector-src/pull/3815#discussion_r1192494161

Reply:
done